app/src/components/AnimalList.js

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+
+import images from "../photos";
+
+const AnimalList = ({ animals }) => {
+  return (
+    <ul>
+      {animals.map(
```

Comment:
Not a big deal, but it’d save you some lines of code if you did...
```suggestion
      {animals.map(animal => { ...
```
and then called each property with `animal.propertyName`